src/Button/Button.tsx

```diff
@@ -3,7 +3,7 @@
 import { colors, ShadedColor } from "../colors";
 import type { Property, SimplePseudos } from "csstype";
 import { base } from "../typography";
-import { ClassNames, jsx, ObjectInterpolation } from "@emotion/core";
+import { ClassNames, jsx, CSSObject } from "@emotion/react";
```

Comment:
`ObjectInterpolation` is no longer re-exported from the `@emotion/react` package. It is available through `@emotion/serialize` (which is a dep of `@emotion/react`), but I didn't want to depend on a child dep, or ask people to include `@emotion/serialize` as a peer dep. That being said, the only difference between `ObjectInterpolation` and `CSSObject` is that `ObjectInterpolation` accepts generics, but in this case we were using a generic of `undefined` anyways, so it shouldn't matter.

Reply:
👍